src/msg/async/rdma/Infiniband.cc

```diff
@@ -890,7 +913,11 @@ void Infiniband::init()
   pd = new ProtectionDomain(cct, device);
   assert(NetHandler(cct).set_nonblock(device->ctxt->async_fd) == 0);
 
-  rx_queue_len = device->device_attr->max_srq_wr;
+  support_srq = cct->_conf->ms_async_rdma_support_srq;
```

Comment:
make srq support related to rdma type?

Reply:
Current iWARP NIC doesn't support SRQ, but it should support it in the near future.